src/types.ts

```diff
@@ -0,0 +1,76 @@
+import { VNode } from 'preact';
+
+export interface Chapter {
+  startTime: number;
+  endTime?: number;
+  name?: string;
+  description?: string;
+}
+
+export interface SEOConfig {
+  baseSegmentsUrl: string;
+}
+
+export enum GroupTypes {
+  mid = 'mid',
+  first = 'first',
+  last = 'last'
+}
+
+export interface CuePoint {
```

Comment:
cue-point interface similar to what we have in navigation plugin. I believe we should move such interfaces to cue-point service rather than keep duplication of interfaces in each plugin that uses cue-points